test/metabase/integrations/slack_test.clj

```diff
@@ -81,20 +82,25 @@
 
     (testing "should be able to fetch channels and paginate"
       (http-fake/with-fake-routes {conversations-endpoint (comp mock-200-response mock-conversations-response-body)}
-        (tu/with-temporary-setting-values [slack-token "test-token"]
-          (is (= (concat (mock-conversations) (mock-conversations))
-                 (slack/conversations-list))))))))
+        (let [expected-result (concat (mock-conversations) (mock-conversations))]
```

Comment:
These tests are kind of weird since we are mocking the response. We're actually testing that a request is formed with the tokens in the correct area (header or body) right? Should we change to test that since that's all we can really check here?

Reply:
Yeah, agree that they're a bit weird, but they do test some other things -- like, given a non-error HTTP response, the functions return the right pieces of data to the caller.

Let me think about if there's any easy ways to clean this up or improve coverage.